.github/workflows/test-action.yml

```diff
@@ -24,6 +24,36 @@ jobs:
       - name: Execute tests
         run: make docker-test
 
+  test-coverage:
+    name: Test coverage
+    runs-on: ubuntu-latest
+    
+    env:
+      MIX_ENV: test
+      NODEROOT: ./node/local
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
can the steps be combined so we use the previous checked out and built version?

Reply:
The checkout is needed before running `./.github/actions/` actions. The `node-setup` was changed to a `mdw-node-setup` though. Would any additional composition be required? There will be also another PR to make dialyzer job faster that will make more explicit what steps can be grouped to be reused.